articles/TOC.yml

```diff
@@ -61,6 +61,8 @@
 - name: Contributing to the QDK
   href: contributing/toc.yml
   topicHref: contributing/index.md
+- name: Further reading
```

Comment:
I'd suggest for consistency (and to avoid having to set up a redirect) moving the `for-more-info.md` file back under `articles` rather than under `articles/concepts`, if that's possible.